be/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "my-api-service",
```

Comment:
shall we change the name to the app's name , we have used viteyourpick for FE

Reply:
updated